src/bundles/peer-locations.test.js

```diff
@@ -30,7 +30,7 @@ jest.mock('hashlru', () => () => ({
 }))
 
 describe('reactPeerLocationsFetch', () => {
-  it('should declare its dependencies', () => {
+  it.skip('should declare its dependencies', async () => {
```

Comment:
@Gozala I wonder if failures here are related to weird behavior I see on Peers screen.
It is unable to fetch geoip data, and requests are for the same CID:

![2021-02-25--02-19-45](https://user-images.githubusercontent.com/157609/109088688-32c6bf00-7710-11eb-95f2-54cf21074761.png)

Perhaps there is a regression somewhere  in  `ipfs.swarm.peers` or  `ipfs.object.*` API, similar to one we had with `ipfs.ls` in https://github.com/ipfs/js-ipfs/issues/3566 ?

Reply:
Hm.. now issue is gone. I think it was just a fluke caused by my node looking for geoip root.